src/main/webapp/index.jsp

```diff
@@ -0,0 +1,90 @@
+<% boolean isUserLoggedIn = (boolean) request.getAttribute("isUserLoggedIn"); %>
```

Comment:
You'll need to add the following before this line to fix Alan's name. This specifies the character set so the UTF-8 character in Alan's last name gets interpreted by the browser correctly.

<%@page contentType="text/html" pageEncoding="UTF-8"%>